includes/discovery/sensors/temperature/beagleboard.inc.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/*
+ * Temperature Sensors for BeagleBoard
+ * Requires snmp extend agent script from librenms-agent
+ */
+
+use Illuminate\Support\Str;
+use LibreNMS\Config;
+
+$type = "beagleboardTemp";
+
+// $oid = 'NET-SNMP-EXTEND-MIB::nsExtendOutLine."beagleboard".';
+// Discovery is fine, but Polling seems to fail with string OID. So snmptranslate,
+//    snmptranslate -On NET-SNMP-EXTEND-MIB::nsExtendOutLine.\"beagleboard\"
+// to (with trailing ., to allow append in loop below),
+$oid = ".1.3.6.1.4.1.8072.1.3.2.4.1.2.11.98.101.97.103.108.101.98.111.97.114.100.";
+for ($sensor = 1; $sensor <= 5; $sensor++) {
+    switch ($sensor) {
+        case "1":
+            $descr = "CPU";
+            break;
+        case "2":
+            $descr = "GPU";
+            break;
+        case "3":
+            $descr = "Core";
+            break;
+        case "4":
+            $descr = "DSP";
+            break;
+        case "5":
+            $descr = "IVA-HD";
+            break;
+    }
+    $value = trim(snmp_get($device, $oid . $sensor, '-Oqve'), '"');
```

Comment:
Can we do a multi here also? Or get the entire $oid maybe?

Reply:
I think I can get walk working here - give me a few 😆

Reply:
OK, pushed - I think this one is fixed ... agreed?

Thanks!